WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -39,6 +39,9 @@ public class BlockEditorScreen: ScreenObject {
      */
     public func enterTextInTitle(text: String) -> BlockEditorScreen {
         let titleView = app.otherElements["Post title. Empty"].firstMatch // Uses a localized string
+        guard titleView.waitForExistence(timeout: 3) else {
+            return self
+        }
```

Comment:
if i'm reading this correctly, if the `titleView` element still does not exist within timeout, it will exit the function and the `tap()` and `typeText()` won't happen. And there's no error thrown in this case.

what do you think about adding an assertion instead? so if it fails, we'll know it's because the title view is not displayed, if it's there then the test will continue.

```suggestion
        XCTAssert(titleView.waitForExistence(timeout: 3), "Title View does not exist!")
```

Reply:
Great point! Agreed!